### PR TITLE
added StatsFuns to imports and require file.

### PR DESCRIPTION
**Pull request by krylea**
While the other pull request has kind of stagnated due to the complicated nature of the distributions at hand, let's at least add these changes, since they are definitely needed no matter what.

### CODE REVIEW

File: src/ConjugatePriors.jl [hunk 1]
```diff
@@ -4,10 +4,19 @@ module ConjugatePriors
 
 using PDMats
 using Distributions
+using StatsFuns
 
 import Base: mean
 import Base.LinAlg: Cholesky
 
+import PDMats:
```
Comment (kleinschmidt) [inline src/ConjugatePriors.jl]: this conflicts with master; you can fix by rebasing your PR on the latest master and resolving the conflict there.

File: src/ConjugatePriors.jl [hunk 2]
```diff
@@ -4,10 +4,19 @@ module ConjugatePriors
...
+    PDMat
+
+import StatsFuns:
+    isApproxSymmmetric,
```
Comment (kleinschmidt) [inline src/ConjugatePriors.jl]: this is in `Distributions` (https://github.com/JuliaStats/Distributions.jl/blob/c13f0b6748b6f6a5353f47f23133d2cdd838820a/src/utils.jl#L118), as is `hasCholesky`, so can you take those two functions from there instead?

```
using Distributions:
    isApproxSymmmetric,
    hasCholesky
```

**Comment by kleinschmidt** [comment]
Also I'm happy to make these changes if you want! And if you enabled edits from maintainers I can do it in this PR as is.

**Comment by krylea** [comment]
If you don't mind, that would be great!

**Comment by kleinschmidt** [comment]
The test failures on 0.7 are because of the PDMats issue so I think this is good to go. Thanks again!

**Pull request merged**